components/notifications-client/builder/ccr.go

```diff
@@ -30,7 +31,7 @@ func ChefClientConverge(url string, run *chef.Run) (*Event, error) {
 		}
 
 		err_desc := err.GetDescription()
-		if err == nil {
+		if err_desc == nil {
```

Comment:
If this value isn't used anywhere, you could do `if err.GetDescription() == nil {` and avoid the variable altogether.

Reply:
It's used below at line 50 of the new file.  Although the use has me thinking this error check isn't needed?  The whole point of the Get() function is that they work on nil and you eventually get a zero-value.  Is it really an error if we didn't get sent a description?  Perhaps we should remove this check completely?

Reply:
Oops, I think I misunderstood this change. To avoid unintentional changes I'd probably go with what you've got now. 😅